Src/UIComponents/Overview/Overview.css

```diff
@@ -0,0 +1,35 @@
+.overview-container{
+    display:flex;
+    flex-wrap: wrap;
+}
+
+.overview-header{
+    flex-basis: 100%;
+    margin-left: 50px;
+}
+
+.overview-header > h1{
+    font-size: 3rem;
+    font-weight: 400;
+    line-height: 1.5;
+}
+
+.overview-header > h2, h2 > span{
+    font-size: 1.5rem;
+    font-weight: 400;
+    line-height: 1.5;
+}
+
+h2 > span{
+    font-weight: 600;
+}
+
+.my-weight-tile{
+    background-color: white;
+    border-radius: 20px;
+}
+
+.tile, .my-weight-tile{
```

Comment:
.tile --> .overview .tile
This kind of selectors could make some problems during importing all .css styles in App.css

Reply:
fixed